### PR TITLE
Update bat-command-reference.adoc [APITEST-1109]

### PR DESCRIPTION
**Pull request by ibottero**

### CODE REVIEW

File: modules/ROOT/pages/bat-command-reference.adoc [hunk 1]
```diff
@@ -72,35 +86,96 @@ Same as the `bat init` command, which sets up a basic project folder. The folder
...
+* `--version`
 +
 Prints to standard output the version numbers of installed BAT and wrapper. If the latest version is not installed, that version is downloaded.
 
-** `environment ls|list|switch`
+* `--wdg`
++
+Enables DataWeave script debugging option.
+
+This command line also offers cloud integration, and to schedule and configure tests and monitors using the Anypoint Platform.
```
Comment (rddheath) [inline modules/ROOT/pages/bat-command-reference.adoc:175]: @ibottero , would you explain further what you mean on line 175?

File: modules/ROOT/pages/bat-command-reference.adoc [hunk 2]
```diff
@@ -110,7 +185,14 @@ The `switch` option enables you to use one of the listed environments. Add the n
 bat environment switch Sandbox
 ----
 
-** `execution ls|list`
+* `exchange pack`
+* `exchange publish`
++
+The `pack` option compress the current folder as zip (ready to be uploaded to Exchange).
++ 
+The `publish` option compress the current folder as zip and upload it to Exchange server. If the asset already exits a new version is incremented so the return zip file is real uploaded one.
```
Comment (rddheath) [inline modules/ROOT/pages/bat-command-reference.adoc:193]: @ibottero 

- Packing as a .zip and publishing to Exchange are new features, aren't they? 
- Why would someone want to publish monitors to Exchange? 
- Where in Exchange are monitors listed?

File: modules/ROOT/pages/bat-command-reference.adoc [hunk 3]
```diff
@@ -142,65 +224,90 @@ bat grant -g=MySecretGroupName -s=MyAlias1:MySecretName1,MyAlias2:MySecretName2,
 +The secret must be created in a secrets group that is in the same environment that you are using in the BAT CLI. To find out which environment you are currently using in the BAT CLI, run the command `bat whoami`. The output contains the ID for the environment. Run the command `bat environment ls` to list the environments that you have access to. Match the ID from the `bat whoami` command with one of the environments listed. If you need to switch to the environment that your secrets group is in, run the command bat environment switch name, where name is the name of the environment.
 --forceUpdate:: Specify this option if API Functional Monitoring lost access to a shared secret and you need to grant access again.
 
-** `location create`
+* `idea`
++
+Create IntelliJ IDEA project files based on the current folder.
```
Comment (rddheath) [inline modules/ROOT/pages/bat-command-reference.adoc:229]: @ibottero , this is a new feature. In which release should it be written up in the release notes?

**Comment by ibottero** [comment]
Thanks Robert

**Review by rddheath — APPROVED** [review]

**Pull request merged**